frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,231 @@
+  });
+
+  it('tests if nooba-core is running', async () => {
+    const pattern = 'noobaa-core-';
+    const pod = getPodData(pods.items, pattern);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-a is running', async () => {
+    const pattern = 'rook-ceph-mon-a-';
+    const pod = getPodData(pods.items, pattern);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-b is running', async () => {
+    const pattern = 'rook-ceph-mon-b-';
+    const pod = getPodData(pods.items, pattern);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mon-c is running', async () => {
+    const pattern = 'rook-ceph-mon-c-';
+    const pod = getPodData(pods.items, pattern);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if rook-ceph-mgr is running', async () => {
+    const pattern = 'rook-ceph-mgr';
+    const pod = getPodData(pods.items, pattern);
+    testPodIsRunning(getPodPhase(pod));
+  });
+
+  it('tests if 5 csi-cephfsplugin are running', async () => {
+    const pattern = 'csi-cephfsplugin-';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(5);
+    podList.forEach((pod) => {
+      testPodIsRunning(getPodPhase(pod));
+    });
+  });
+
+  it('tests if 5 csi-rbdplugin are running', async () => {
+    const pattern = 'csi-rbdplugin-';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(5);
+    podList.forEach((pod) => {
+      testPodIsRunning(getPodPhase(pod));
+    });
+  });
+
+  it('tests if 2 rook-ceph-mds-ocs-storagecluster-cephfilesystem pods are running', async () => {
+    const pattern = 'rook-ceph-mds-ocs-storagecluster-cephfilesystem';
+    const podList = getPodData(pods.items, pattern);
+    expect(podList.length).toBe(2);
+    podList.forEach((pod) => {
+      testPodIsRunning(getPodPhase(pod));
+    });
+  });
+
+  it('tests if all ceph-rbd, cephfd, noobaa storage classes are shown', async () => {
+    const pattern1 = 'ocs-storagecluster-ceph-rbd';
+    const pattern2 = 'ocs-storagecluster-cephfs';
+    const pattern3 = 'noobaa.io';
+    await goToStorageClasses();
+    await browser.wait(until.visibilityOf(filterInput));
+    await filterInput.sendKeys(pattern1);
+    await browser.sleep(200);
+    const rdbClass = await getDataFromRowAndCol(0, 1, podNameFilter);
+    expect(rdbClass).toEqual(pattern1);
+    await filterInput.clear();
+    await browser.wait(until.visibilityOf(filterInput));
+    await filterInput.sendKeys(pattern2);
+    await browser.sleep(200);
+    const fsClass = await getDataFromRowAndCol(0, 1, podNameFilter);
+    expect(fsClass).toEqual(pattern2);
+    await filterInput.clear();
+    await browser.wait(until.visibilityOf(filterInput));
+    await filterInput.sendKeys(pattern3);
+    await browser.sleep(200);
+    const noobaaClass = await getDataFromRowAndCol(0, 1, podNameFilter);
+    expect(noobaaClass.includes(pattern3)).toBe(true);
+  });
```

Comment:
We also need to add tests for verifying ocs label here.

Reply:
ack

Reply:
can you provide the exact label?

Reply:
'cluster.ocs.openshift.io/openshift-storage'

Reply:
Done